### PR TITLE
New package: Roblox.RobloxStudio version 1.6.3.206

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112003&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by upintheairsheep** [comment]
> image

Again, please.

**Comment by ItzLevvie** [comment]
[Policy] Interactive-Only-Installer

**Comment by stephengillie** [comment]
> Again, please.

It's unchanged - Automatic Validation ended with:
image

File: manifests/r/Roblox/RobloxStudio/1.6.3.206/Roblox.RobloxStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.6.3.206
+Installers:
+- InstallerUrl: https://setup.rbxcdn.com/RobloxStudioLauncher.exe
+  Architecture: x86
+  InstallerType: exe
+  InstallerSha256: D39D8F3E3C98F9059BF359A9C2C56C5565DD71F2F9913D399500A4FCBB9A3CC0
+  InstallerSwitches:
+    Silent: /install /silent
+    SilentWithProgress: /install /silent
```
Comment (stephengillie) [inline manifests/r/Roblox/RobloxStudio/1.6.3.206/Roblox.RobloxStudio.installer.yaml:13]:
```suggestion
    Silent: /install silent
    SilentWithProgress: /install silent
```
I'm guessing that the 2nd `/` or `-` might be causing the error.

**Pull request closed without merging**